p2pclient/go/p2pclient.go

```diff
@@ -30,7 +30,7 @@ type Client struct {
 // NewClient creates a new libp2p daemon client, connecting to a daemon
 // listening on a unix socket at controlPath, and establishing an inbound socket
 // at listenPath.
-func NewClient(controlPath, listenPath string) (*Client, error) {
+func NewClient(controlPath string, listenPath string) (*Client, error) {
```

Comment:
please don't change this, it's idiomatic go to write arguments with common type like this.

Reply:
10-4 Thanks for the clarification